### PR TITLE
Translate Japanese comments to English (Fixes #50)

### LINKED ISSUE #50: Internationalise codebase
There are lots of Japanese comments in the source files which prevent international contributors to understand it.
For example:
// 受信スレッド生成
These should be translated to English or removed from the code.

### PR DESCRIPTION
**Pull request by Zotyamester**
#50 
All comments were translated via Google Translate.

### CODE REVIEW

File: src/FTPClient.cpp [hunk 1]
```diff
@@ -35,7 +35,7 @@ TFTPClient::TFTPClient()
 		return;
 	}
 	
-	// 受信スレッド生成
+	// Create receive thread
```
Comment (korli) [inline src/FTPClient.cpp]: receiving thread

File: src/FTPClient.cpp [hunk 2]
```diff
@@ -205,7 +205,7 @@ const char *TFTPClient::StrReply() const
 	return fStrReply.String();
 }
 
-// 受信スレッド関数
+// Receive thread function
```
Comment (korli) [inline src/FTPClient.cpp]: receiving thread

File: src/FTPWindow.cpp [hunk 3]
```diff
@@ -883,12 +883,12 @@ void TFTPWindow::Chmod()
 	if (strpermField->String()[8] != '-') mode |= 02;
 	if (strpermField->String()[9] != '-') mode |= 01;
 
-	// permission 変更窓を表示
+	//
```
Comment (korli) [inline src/FTPWindow.cpp]: This is now blank?

File: src/OAEdit.cpp [hunk 4]
```diff
@@ -18,8 +18,8 @@ TOAFilter::~TOAFilter()
 	delete OAMessenger;
 }
 
-//	BTextControl の子 BView である BTextView に送られる BMessage をフィルタする。
-//	ENTERキーと↓、↑を横取りして、次または前の BView に MakeFocus する。
+// Filters the BMessage sent to BTextView, a child BView of BTextControl.
+// Intercept the ENTER key and ↓, し て, and MakeFocus to the next or previous BView.
```
Comment (korli) [inline src/OAEdit.cpp]: Intercept the ENTER key and ↓, ↑, and MakeFocus to the next or previous BView.

File: src/FTPWindow.cpp (hunk 3 above)
Comment (Zotyamester) [reply src/FTPWindow.cpp]: Receive thread function

File: src/OAEdit.cpp [hunk 5]
```diff
@@ -83,8 +83,8 @@ status_t TOAEdit::Archive(BMessage *archive, bool deep) const
 
 void TOAEdit::AttachedToWindow()
 {
-// 中にいるBTextViewにメッセージフィルタをかまして
-// ENTERキー、および矢印キーを監視できるようにする。
+// Bite the message filter on the BTextView inside.
```
Comment (korli) [inline src/OAEdit.cpp]: This needs a better translation.

File: src/RemoteFileView.cpp [hunk 6]
```diff
@@ -18,7 +18,7 @@ TStringColumn::~TStringColumn()
 {
 }
 
-// ディレクトリ優先のファイル名ソート関数 (改造 BColumnListView が必要)
+// Directory-first file name sorting function (remodeled BColumnListView required)
```
Comment (korli) [inline src/RemoteFileView.cpp]: (Requires modified BColumnListView)

File: src/OAEdit.cpp (hunk 5 above)
Comment (Zotyamester) [reply src/OAEdit.cpp]: If you know Japanese, then sure, go ahead. I can only go by what the translator tells me.

File: src/FTPClient.cpp (hunk 1 above)
Comment (Zotyamester) [reply src/FTPClient.cpp]: Then it should be rather "Create receiver thread"

File: src/TCPStream.cpp [hunk 7]
```diff
@@ -17,21 +17,21 @@ TTCPStream::TTCPStream(BDataIO *dataIO, struct sockaddr_in *addr,
 		return;
 	}
 	
-	// 送受信バッファ確保
+	// Send / receive buffer reservation
```
Comment (korli) [inline src/TCPStream.cpp]: Send / receive buffer allocation

**Comment by scottmc** [comment]
Closing this one as there is another.

**Pull request closed without merging**